packages/adminrouter/extra/src/includes/server/common.conf

```diff
@@ -1,3 +1,10 @@
+# Group: Status
```

Comment:
s/Status/Metrics/ ?

Reply:
I renamed it to `Status` to no conflict with the DC/OS `/v0/metrics` API.